### PR TITLE
Add option plassen

### PR DESCRIPTION
**Pull request by MauriceBuiten**
[Review on Reviewable](https://reviewable.io/reviews/udacity/create-your-own-adventure/1821)

### CODE REVIEW

**Comment by cbuckey-uda** [comment]
@casey-collab, please review this

File: dutch/bos/ontvoering/meehelpen/klus/plassen.md [hunk 1]
```diff
@@ -0,0 +1 @@
+Zodra je de bosjes bereikt hebt....
```
Comment (casey-collab) [inline dutch/bos/ontvoering/meehelpen/klus/plassen.md]: It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by cbuckey-uda** [comment]
@casey-collab, please take another look

File: dutch/bos/ontvoering/meehelpen/klus/plassen.md (hunk 1 above)
Comment (casey-collab) [inline dutch/bos/ontvoering/meehelpen/klus/plassen.md]: It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.

File: dutch/bos/ontvoering/meehelpen/meehelpen.md [hunk 2]
```diff
@@ -5,4 +5,6 @@ Een sterke man kunnen ze goed gebruiken omdat de lijkzak erg zwaar is.
 Ze houden je goed in de gaten, hoe red je je nou uit deze situatie?
 
 Je maakt een slechte grap over de vorige man die je zo versleept hebt en hoe 
-goed die [klus](klus/geld-verdienen.md) betaalde.
\ No newline at end of file
+goed die [klus](klus/geld-verdienen.md) betaalde.
+
+Je zegt dat je even moet [plassen](plassen/watnu.md), en je loopt richting de bosjes.
```
Comment (casey-collab) [inline dutch/bos/ontvoering/meehelpen/meehelpen.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: dutch/plassen/watnu.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
+Zodra je de bosjes bereikt hebt...
```
Comment (casey-collab) [inline dutch/plassen/watnu.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

File: dutch/bos/ontvoering/meehelpen/klus/plassen.md (hunk 1 above)
Comment (casey-collab) [inline dutch/bos/ontvoering/meehelpen/klus/plassen.md]: This page is also unreachable.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

File: dutch/bos/ontvoering/meehelpen/meehelpen.md (hunk 2 above)
Comment (MauriceBuiten) [inline dutch/bos/ontvoering/meehelpen/meehelpen.md]: I moved the directory back to where it was in the first place, and now all the links are working again, I personally checked them all. Thank you.

**Comment by cbuckey-uda** [comment]
@cas

**Comment by cbuckey-uda** [comment]
I'm so sorry cas, I keep doing that! @casey-collab, please take another look.

**Comment by casey-collab** [comment]
Looks great! :icecream:

**Comment by cas** [comment]
LOL np.  Take care.
-Cas

**Comment by casey-collab** [comment]
Unfortunately, this pull request conflicts with another change. Could you merge the master branch of the original repository into your branch, resolve the merge conflicts, and update the pull request?

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Pull request merged**